From 0d22fb7ce2d82a901d0b0553affbb8ccff49fa03 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Tue, 5 Jun 2012 11:23:14 +0800
Subject: [PATCH 1586/2463] ENGR00212322-3 usb: add one platform data entry to
 store anatop base address

It is used to usb charger detect, the charger detection process need
to access anatop register.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 include/linux/fsl_devices.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 5437d3a..8c41b23 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -145,6 +145,8 @@ struct fsl_usb2_platform_data {
 	enum usb_wakeup_event	wakeup_event;
 	u32		pmflags;	/* PM from otg or system */
 
+	void __iomem *charger_base_addr; /* used for i.mx6 usb charger detect */
+
 	/* register save area for suspend/resume */
 	u32		pm_command;
 	u32		pm_status;
-- 
1.7.7.4

